### PR TITLE
fix: fetch bundle with bundle identifier SQSERVICES-1031

### PR DESCRIPTION
**Pull request by johnxnguyen**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/SQSERVICES-1031" title="SQSERVICES-1031" target="_blank">BugSQSERVICES-1031</a>  [iOS] Release build crashes
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

See https://github.com/wireapp/wire-ios-data-model/pull/1291

### Causes

See https://github.com/wireapp/wire-ios-data-model/pull/1291

### Solutions

This is another attempt to get the correct Bundle (via the bundle identifier). It seems that fetching the [bundle via a class type](https://developer.apple.com/documentation/foundation/bundle/1417717-init) is not working correctly.

### Testing

Can only be tested in the testflight build.

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Review by typfel — APPROVED** [review]

**Review by KaterinaWire — APPROVED** [review]

**Comment by johnxnguyen** [comment]
release patch

**Comment by johnxnguyen** [comment]
release patch

File: Resources/Configurations/version.xcconfig [hunk 1]
```diff
@@ -1,2 +1,2 @@
-MAJOR_VERSION = 272
-CURRENT_PROJECT_VERSION = 272.0.2
+MAJOR_VERSION = 277
+CURRENT_PROJECT_VERSION = 277.0.3
```
Comment (billypchan) [inline Resources/Configurations/version.xcconfig:2]:
```suggestion
MAJOR_VERSION = 272
CURRENT_PROJECT_VERSION = 272.0.2
```

**Comment by billypchan** [comment]
release patch

**Comment by billypchan** [comment]
release patch

**Comment by billypchan** [comment]
release patch

**Pull request merged**